docs/sample_inventories/scram_kraft.yml

```diff
@@ -0,0 +1,49 @@
+---
+##
+## The following is an example inventory file of the configuration required for setting up Confluent Platform with Kraft Controller with SCRAM.
+## This will set up controller-controller and controller-broker communication via plain and other communication via scram.
```

Comment:
```suggestion
## This will set up controller-controller and controller-broker communication via plain and other communication, including inter-broker via scram.
```